libraries/app/api.cpp

```diff
@@ -453,6 +453,37 @@ namespace graphene { namespace app {
        return result;
     }
     
+    vector<operation_history_object> history_api::get_account_history_operations( account_id_type account, 
+                                                                       int operation_id,
+                                                                       operation_history_id_type start, 
+                                                                       operation_history_id_type stop,
+                                                                       unsigned limit) const
+    {
+       FC_ASSERT( _app.chain_database() );
+       const auto& db = *_app.chain_database();
+       FC_ASSERT( limit <= 100 );
```

Comment:
if you move the assertion by -1 line .. you can spare instanciating the database if the limit is to high :D